ArmaForces.Boderator/ArmaForces.Boderator.BotService/Discord/DependencyInjection/IDiscordServiceBuilder.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace ArmaForces.Boderator.BotService.Discord.DependencyInjection
+{
+    internal interface IDiscordServiceBuilder
+    {
+    }
+}
```

Comment:
Missing newline at end of file.
```suggestion
}

```